app/channels/room_channel.rb

```diff
@@ -0,0 +1,13 @@
+class RoomChannel < ApplicationCable::Channel
```

Comment:
What will correspond to a "room"? Are we aiming for notifications of comments to start with? Is each node a "room" and comments are messages in the room? Maybe we should make a "NodeChannel" instead, what do you think?

Reply:
There will be multiple channels for multiple features, but I was thinking of making a room_channel which will be connected to all the users irrespective, in the case we want to use it in the future to send some notifications to all the users even they are logged in or not.
And I am going to make a differnt channel for a different feature, for example, node_channel, like_channel etc.
What do you say?

Reply:
What are your thoughts about this @jywarren ?

Reply:
Ah, my apologies for missing this!!! Yes, this sounds good; can you add this as a comment at the top of this file, just so there's some context?